openlibrary/templates/type/author/view.html

```diff
@@ -163,16 +163,23 @@ <h6 class="collapse black uppercase">$label</h6>
             </div>
 
         <div class="section">
-            <h3>$:_('Links <span class="gray small sansserif">(outside Open Library)')</span></h3>
-
-            $if page.links or page.remote_ids or (page.wikipedia and page.wikipedia.startswith("http")):
-                <ul class="booklinks sansserif">
+            <h3>$:_('ID Numbers')</h3>
```

Comment:
quick q - why does this project do strings like this? Is it a web.py convention? Not very familiar.

```
<h3>$:_('ID Numbers')</h3>
```

Thanks in advance if anyone has time to answer!

Reply:
I think it has to do with i18n/translation s

Reply:
There are two things happening here.  Any time you see `_('Some string')` in our `.html` files, the string within the parenthesis can be replaced if there is a translation for it (more details about this can be found [here](https://github.com/internetarchive/openlibrary/tree/master/openlibrary/i18n#i18n-contributors-guide)).

The `$` is used to specify a Python expression in Templetor, web.py's templated system.  This does HTML encoding by default.  If raw HTML is need in a string, encoding can be turned off by using a colon, like this `$:` (more info [here](https://webpy.org/docs/0.3/templetor#filtering)).

As an example, the "Links (outside of Open LIbrary)" heading would be rendered like this if the encoding wasn't turned off:
![Screenshot from 2021-08-16 21-45-35](https://user-images.githubusercontent.com/28732543/129650725-114e47c0-10ea-4771-95a5-e912ad1fb57c.png)

In the case of the "ID Numbers" heading, the colon isn't needed.
```suggestion
            <h3>$_('ID Numbers')</h3>
```